ts/screens/messages/MessagesHomeScreen.tsx

```diff
@@ -211,7 +214,9 @@ class MessagesHomeScreen extends React.Component<Props> {
       .map(
         _ =>
           _.length < 3 ? (
```

Comment:
You should avoid using magic number, define it as a const

Reply:
Done. In the commit f3a51b5